dev/bots/download_goldctl.ps1

```diff
@@ -0,0 +1,14 @@
+$url= "https://storage.googleapis.com/chrome-infra/depot_tools.zip"
```

Comment:
Updated to work on Cirrus. Verified in earlier commit by using a mock-post-submit state in pre-submit to run Skia tests.